hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/navBar/navBar.tsx

```diff
@@ -111,11 +111,23 @@ class NavBar extends React.Component<INavBarProps> {
             <span>Containers</span>
             <Link to='/Containers'/>
           </Menu.Item>
-          <Menu.Item key='/Insights'>
-            <Icon type='bar-chart'/>
-            <span>Insights</span>
-            <Link to='/Insights'/>
-          </Menu.Item>
+          <Menu.SubMenu key='/In'
```

Comment:
Nit: Can we rename the key to be more descriptive?

Reply:
@devabhishekpal 
Completed Change in latest commit.